webui/app.py

```diff
@@ -176,7 +176,25 @@ def index():
             # Extract the .tar to a temp directory
             img_temp_path = tempfile.mkdtemp()
             with tarfile.open(upload_filepath) as tar:
-                tar.extractall(img_temp_path)
+               def is_within_directory(directory, target):
```

Comment:
This seems to have come from the CVE fix PR - please rebase and remove from this PR

Reply:
Merged with latest & I do not see this file in the updated review. 8 files are displayed & app.py is not one among them.